database/migrations/2022_12_08_152100_create_auth0_clients_table.php

```diff
@@ -19,6 +21,9 @@ public function up(): void
             $table->softDeletes();
             $table->timestamps();
         });
+
+        // Migrate old clients that don't have an id as a primary key
+        DB::statement('UPDATE auth0_clients SET id = UUID() WHERE id \'\'');
```

Comment:
No need for this, because when creating a table there can't be any content. We will do a delete of the existing tables anyway.